spec.html

```diff
@@ -324,7 +322,7 @@
       <h1>BigInt::signedRightShift (_x_, _y_)</h1>
       <p>The abstract operation BigInt::signedRightShift with arguments _x_ and _y_ of type BigInt:</p>
       <emu-alg>
-        1. Return ? BigInt::leftShift(_x_, -_y_).
+        1. Return BigInt::leftShift(_x_, -_y_).
```

Comment:
Nit: Rather than eliminating the `?` , better to replace it with `!` because the definition of leftShift may throw for certain types (though we assert it won't). For example, for Number, it will call ToInt32, which could throw from ToNumber. That won't throw, since it's already a Number, however.